### DISCUSSION TITLE
能否提供国内访问热度前xx万的域名列表？

### DISCUSSION THREAD

**Discussion by yyysuo** [kkkgo/PaoPao-Pref#9] [提问和帮助]
不一定是国内域名，是在中国大陆被访问的最频繁的域名 。

**Comment by kkkgo** [comment]
没有公开数据来源。全球域名来自于思科公开数据。
https://s3-us-west-1.amazonaws.com/umbrella-static/index.html

**Reply by yyysuo** [reply]
> 没有公开数据来源。全球域名来自于思科公开数据。 https://s3-us-west-1.amazonaws.com/umbrella-static/index.html

发现cf也有类似的东西
https://radar.cloudflare.com/domains

**Reply by yyysuo** [reply]
root@QWRT:/tmp# ./paopao-perf -server 127.0.0.1 -port 5454 -pc 1
Percentage : 1 %
Total Line : 8461
Concurrency Limit: 10
Timeout: 5s
Sleep: 1.5s
DNS Server: 127.0.0.1
DNS Port: 5454
Nslookup 8481/8461 domains[100.2364%]. Succ rate:99.83%. Avg time: 0.154785 seconds. Est time: 00:00:-3.
Done. Processed 8481/8461 domains in 1314.37 seconds.

**Reply by kkkgo** [reply]
cloudflare提供的是主域名，对DNS测试记录作用不大。

**Reply by yyysuo** [reply]
> 试记录作用不

了解了，目前正在跑全部的80多万域名，按默认的配置，得跑35个小时 。

**Comment by yyysuo** [comment]
root@QWRT:/tmp# ./paopao-perf -server 127.0.0.1 -port 5454
Percentage : 100 %
Total Line : 846158
Concurrency Limit: 10
Timeout: 5s
Sleep: 1.5s
DNS Server: 127.0.0.1
DNS Port: 5454
Nslookup 846158/846158 domains[100.0000%]. Succ rate:99.23%. Avg time: 0.164006 seconds. Est time: 00:00:00.
Done. Processed 846158/846158 domains in 138779.55 seconds.


我没刷paopaodns，刷mosdns的缓存，把它给刷炸了.............    刷了10个小时保存cache的时候就报错，刷到30多个小时mosdns直接崩溃重启了..........
![image](https://github.com/user-attachments/assets/7d5ef0ab-cd58-4292-957e-8ec9aff34f13)

![image](https://github.com/user-attachments/assets/bd922a5c-7305-4ff1-a5b9-96842f038a86)

**Reply by kkkgo** [reply]
看你描述应该是用了mosdns的lazy cache之类的功能。paopaodns的持久缓存实际上是由unbound和redis管理的。
